.github/workflows/build_wheels.yml

```
@@ -53,6 +76,7 @@ jobs:
           pip install bodo
       - name: Test
         run: |
+          set -exo pipefail
```

```suggestion
          set -eo pipefail
```
Had issues with this printing tokens

---

See above